sdk/core/azure-core/src/main/java/com/azure/core/implementation/serializer/HttpResponseBodyDecoder.java

```diff
@@ -63,8 +63,7 @@
                         } catch (IOException | MalformedValueException ignored) {
```

Comment:
The name is misleading because this is not actually ignored. Should be error or ex or e, etc.